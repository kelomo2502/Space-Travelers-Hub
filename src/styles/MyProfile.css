.Profile-container {
  padding: 5%;
}

.Section {
  display: grid;
  grid-template-columns: 50% 50%;
  gap: 28px;
  justify-content: center;
}

.Content {
  border: 1px solid lightgray;
  border-radius: 4px;
  padding: 16px 0;
}

.Content p {
  border-bottom: 1px solid lightgray;
  padding: 0 24px;
}
