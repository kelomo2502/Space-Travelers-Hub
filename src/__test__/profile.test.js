import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configurestore';
import MyProfile from '../components/profile';

it('Renders MyProfile component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
