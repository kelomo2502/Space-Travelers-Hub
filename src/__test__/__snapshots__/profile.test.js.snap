// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders MyProfile component correctly 1`] = `
<div
  className="Profile-container"
>
  <h2>
    My Profile
  </h2>
  <div
    className="Section"
  >
    <div
      className="Missions"
    >
      <h4>
        My Mission
      </h4>
      <div
        className="Content"
      >
        <p>
          No Missions
        </p>
      </div>
    </div>
    <div
      className="Rockets"
    >
      <h4>
        My Rockets
      </h4>
      <div
        className="Content"
      >
        <p>
          No reservation!
        </p>
      </div>
    </div>
  </div>
</div>
`;
